Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -67,12 +85,12 @@ public static LootLockerConfig current
         public string domainKey;
         [HideInInspector]
         public int gameID;
-        public string game_version = "1.0";
+        public string game_version = "1.0.0.0";
```

Comment:
Standard semver is 3 "decimals" only so I think the 4th is unnecessary (though since it's really only a string, it doesn't really matter). Actually, if we should have any default for game version, shouldn't it be something like 0.0.1 since it's the initial value?

Reply:
We say this; 
game_version being the current version of the game in the format 1.2.3.4 (the 3 and 4 being optional but recommended)
in the API documentation https://ref.lootlocker.com/game-api/#authentication-request
So thought that it didn't hurt to have the same in the SDK, but we can remove it.